app/views/providers/submitted_applications/show.html.erb

```diff
@@ -43,6 +43,51 @@
           ) %>
     </section>
 
+    <% if Setting.allow_multiple_proceedings? %>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h2 class="govuk-heading-m"><%= t '.section_delegated.heading' %></h2>
+        </div>
+      </div>
+    <% else %>
+      <h2 class="govuk-heading-m"><%= t '.section_delegated.heading' %></h2>
+    <% end %>
```

Comment:
how come the heading here is formatted differently based on mul proc/no mul proc?

Reply:
Good question... it's duplicated from `check_provider_answers/_shared` and should _definitely_ be reviewed as part of the future CYA refactor